### PR TITLE
filter testimonies: twig index + adminController index + methode test…

### PR DESCRIPTION
**Pull request by Vannou28**
…imoniesManager

### CODE REVIEW

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
guess why ;)

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Review by Mcbess-san — DISMISSED** [review]

File: src/View/Admin/Home/Component/_admin_testimony.html.twig [hunk 1]
```diff
@@ -1,8 +1,21 @@
-<main id="adminTestimonies" class="container">
+<section id="adminTestimonies" class="container">
     <h2 class="mt-5 pb-2">
         Liste des témoignages
-    </h2>
-    <div class="table-responsive">
+    </h2>                      
+    <label for="selectTestimonies">Filtre d'affichage des témoignages</label>
```
Comment (LucyMerlier) [inline src/View/Admin/Home/Component/_admin_testimony.html.twig:5]: label should be inside of form

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**